docs/examples/using-defaults.md

```diff
@@ -0,0 +1,269 @@
+---
```

Comment:
I'd like to lint this for Writers' Toolkit style in case people pattern match against this but we can do that in a follow up PR.